packages/core/package.json

```diff
@@ -2,9 +2,9 @@
   "name": "pcln-design-system",
   "version": "2.0.0-beta.5",
   "description": "Priceline Design System",
-  "main": "dist/index.js",
+  "main": "dist/index.esm.js",
```

Comment:
Just double checking. Would we have support concern `esm` in dist build?

Reply:
ES modules have to be in dist for treeshaking to work correctly with webpack AFAIK